ypy_websocket/django_channels/yroom_storage.py

```diff
@@ -0,0 +1,226 @@
+import time
+from typing import Optional
+
+import redis.asyncio as redis
+import y_py as Y
+
+
+class BaseYRoomStorage:
+    """Base class for YRoom storage.
+
+    This class is responsible for storing, retrieving, updating and persisting the Ypy document.
+
+    Each Django Channels Consumer should have its own YRoomStorage instance, so each consumer is connected to the same document.
```

Comment:
This sentence is a bit confusing to me. It would make more sense to me as
```suggestion
    Each Django Channels Consumer should have its own YRoomStorage instance, although all consumers and rooms with the same room name will be connected to the same document in the end.
```
or something like that